test/e2e/autoscale_test.go

```diff
@@ -259,7 +272,18 @@ func assertScaleDown(ctx *testContext) {
 }
 
 func TestAutoscaleUpDownUp(t *testing.T) {
-	ctx := setup(t)
+	ctx := setup(t, 8080, "autoscale")
+	stopChan := DiagnoseMeEvery(15*time.Second, ctx.clients, ctx.logger)
+	defer close(stopChan)
+	defer tearDown(ctx)
+
+	assertScaleUp(ctx)
+	assertScaleDown(ctx)
+	assertScaleUp(ctx)
+}
+
+func TestAutoscaleUpDownUpUserPort(t *testing.T) {
```

Comment:
Even though I believe the AutoscaleUpDownUpTest is the most comprehensive test in our suite, I think it may be a bit overkill for this purpose.  It is already a bit flakier than other tests (since it touches most features), so running it twice may be making that worse (we do see a lot more failures in this PR).  It also uses more resources than other tests.  

Can we please just do HelloworldUserPort instead?  Thanks

Reply:
ok, no problem